cluster-provision/k8s/fetch-latest-cdi.sh

```diff
@@ -2,24 +2,60 @@
 
 set -euo pipefail
 
-provision_dir="$1"
-[ ! -d $provision_dir ] && echo "directory $provision_dir does not exist!" && exit 1
-[ ! -d $provision_dir/manifests ] && echo "directory $provision_dir/manifests does not exist!" && exit 1
+force=
+while getopts ":f" opt; do
+    case "${opt}" in
+    f)
+        force=true
+        shift
+        ;;
+    \?)
+        usage
```

Comment:
@dhiller I think the implementation of `usage` is missing

Reply:
Good catch, thanks!